backend/queue-service/src/main/kotlin/com/ticketqueue/queue/scheduler/QueueCleanupScheduler.kt

```
@@ -0,0 +1,61 @@
+    private val queueService: QueueService
+) {
+
+    private val logger = KotlinLogging.logger {}
+
+    @Scheduled(cron = "0 0 3 * * *", zone = "Asia/Seoul")
+    fun executeCleanup() {
+        logger.info { "Queue cleanup batch started" }
+
+        val scheduleIds = try {
+            eventServiceClient.getEndedScheduleIds().scheduleIds
+        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is too generic. Prefer catching specific exceptions to the case that is currently handled.

(detekt.exceptions.TooGenericExceptionCaught)